src/ucp/api/ucp.h

```diff
@@ -391,6 +392,32 @@ void ucp_config_print(const ucp_config_t *config, FILE *stream,
 
 /**
  * @ingroup UCP_CONTEXT
+ * @brief Get UCP library version.
+ *
+ * This routine returns the UCP library version.
+ *
+ * @param [out] major_version       Filled with library major version.
+ * @param [out] minor_version       Filled with library minor version.
+ * @param [out] release_number      Filled with library release number.
+ */
+void ucp_get_version(unsigned *major_version, unsigned *minor_version,
+                     unsigned *release_number);
+
+
+/**
+ * @ingroup UCP_CONTEXT
+ * @brief UCP context initialization with particular API version.
```

Comment:
We have to hide this one in doxygen and explicitly document it as an internal function.